conda/meta.yaml

```diff
@@ -36,6 +36,7 @@ test:
     - boto3
```

Comment:
can you update the test requirements to use the new `ts-conda-build` meta package? should be something like:

```
    - ts-conda-build
    - ts-ddsconfig
    - ts-dds
    - ts-idl
```